Tests/Foundation/Tests/TestFileHandle.swift

```diff
@@ -111,7 +111,14 @@ class TestFileHandle : XCTestCase {
 #else
         var fds: [Int32] = [-1, -1]
         fds.withUnsafeMutableBufferPointer { (pointer) -> Void in
-            pipe(pointer.baseAddress)
```

Comment:
Why don't we [simply force-unwrap this, as I did in my earlier pull](https://github.com/apple/swift-corelibs-foundation/pull/4850/files#diff-35d3bfa0715ea3ee3bdaeeb75d5b6777108198ac6c9da64a65eff677b08bb6e7R114)? If you agree, simply remove this change from this pull and I will get that earlier change from my pull merged instead.